packages/css/src/plugins/expand-shorthands/index.tsx

```diff
@@ -59,31 +59,38 @@ const shorthands: Record<string, ConversionFunction> = {
 /**
  * PostCSS plugin that expands shortform properties to their longform equivalents.
  */
-export const expandShorthands = plugin('expand-shorthands', () => {
-  const filter = new RegExp(Object.keys(shorthands).join('|'));
-
-  return (root) => {
-    root.walkDecls(filter, (decl) => {
-      const valueNode = parse(decl.value);
+export const expandShorthands = (): Plugin => {
```

Comment:
super nit: Again with the returning arrow function nagging sorry. There are couple more in the files below. Are we avoiding them as code style policy?